includes/class-wp-job-manager.php

```diff
@@ -572,8 +572,6 @@ public function frontend_scripts() {
 			]
 		);
 
-		Job_Overlay::instance()->init_dashboard_overlay();
```

Comment:
It's already called from the shortcode class [here](https://github.com/Automattic/WP-Job-Manager/blob/f1d376fddb2853b2bd41cceffce52245b06f6a84/includes/class-job-dashboard-shortcode.php#L104), this was just a mistake earlier. (And a bug, because we outputted the <dialog> element with the same ID multiple times, which broken the JS)